R/utils-html.R

```diff
@@ -193,7 +193,7 @@ getComments <- function(comments) {
 #' @param inQualifiers data frame of filtered (for SVP) or all (for SRS) qualifiers.
 #' @return list of deduplicated qualifiers with column names.
 formatQualifiersTable <- function(inQualifiers) {
-    if (isEmptyOrBlank(inQualifiers) || nrow(inQualifiers) == 0) return()
+    if(isEmptyOrBlank(inQualifiers) || (isEmptyVar(inQualifiers))) return()
```

Comment:
yes, I see that this is pretty close to being what we need how you like so I'll get it to work with one return. Thanks!